test/presenters/ht_registration_presenter_test.rb

```diff
@@ -89,3 +89,82 @@ def setup
     assert_match "/ht_registrations/#{reg.id}", reg.cancel_path
   end
 end
+
+class HTRegistrationPresenterENVTest < ActiveSupport::TestCase
+  def setup
+    @inst = create(:ht_institution, inst_id: "test",
+      allowed_affiliations: "faculty@default.invalid;staff@default.invalid")
```

Comment:
We would normally expect this to be `^(faculty|staff)@default.invalid`

Reply:
Aaron Testuser Elkiss now shows OK badge in testing with latest patch.